src/test/resources/job_response.json

```diff
@@ -0,0 +1,27 @@
+{
+    "info": [
+        {
+            "StartTime": "2015, Mar 04 19:28:17.363806",
+            "Target": "*",
+            "Target-type": "glob",
+            "User": "johnny",
+            "Arguments": [],
+            "Function": "test.ping",
+
+            "jid": "20150304192817363806",
+            "Minions": [
+                "blackbox"
+            ],
+            "Result": {
+                "blackbox": {
+                    "return": true
+                }
+            }
+        }
+    ],
+    "return": [
+        {
+            "blackbox": true
+        }
+    ]
+}
```

Comment:
yes, just added it.

2015-03-15 16:19 GMT+01:00 Kevin Walter <notifications@github.com>:

> In src/test/resources/job_response.json
> <https://github.com/SUSE/saltstack-netapi-client-java/pull/45#discussion_r26448658>
> :
>
> > +            "Minions": [
> > +                "blackbox"
> > +            ],
> > +            "Result": {
> > +                "blackbox": {
> > +                    "return": true
> > +                }
> > +            }
> > +        }
> > +    ],
> > +    "return": [
> > +        {
> > +            "blackbox": true
> > +        }
> > +    ]
> > +}
>
> missing newline here i'm not sure if checkstyle catches this yet for
> ressources (maybe something to add in #53
> <https://github.com/SUSE/saltstack-netapi-client-java/pull/53>)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SUSE/saltstack-netapi-client-java/pull/45/files#r26448658>
> .
>